Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/sorear/smetamath-rs"
 clap = "2.5.2"
 filetime = "0.1.10"
 fnv = "1.0.2"
+regex = "0.1.71"
```

Comment:
Not that I know offhand, though I haven't really looked.  I just remember the i.r-l.o discussions where the regex crate authors were using "faster than GNU grep on large files" as their primary performance endpoint.